packages/story-editor/src/components/floatingMenu/elements/shared/text.js

```diff
@@ -19,13 +19,19 @@
  */
 import PropTypes from 'prop-types';
 import { memo } from '@googleforcreators/react';
+import styled from 'styled-components';
 import { ContextMenuComponents } from '@googleforcreators/design-system';
 
+const Button = styled(ContextMenuComponents.MenuButton)`
+  font-size: 14px;
```

Comment:
I was a bit confused here too. Cause `BUTTON_SIZE.SMALL` is not the same as `PRESET_SIZES.SMALL`. The `PRESET_SIZES` under typography has `label` listed as font size 14 but the small button was getting size 12. But upon further investigation it looks like the `ContextMenuComponent.MenuButton` is actually using paragraph, which makes sense since the text here isn't a label. in `contextMenu -> components -> styles -> menuItemStyles`
```themeHelpers.expandPresetStyles({
      preset:
        theme.typography.presets.paragraph[
          THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL
        ],
      theme,
```

That being said, even if I try to update the `attrs` for `ContextMenuComponent.MenuButton` to `MEDIUM` it doesn't get overridden. And it didn't seem like the right move to try and mess with context menu styling, so I am not sure _how_ I can use any of the theming here 🤷🏻‍♀️

Reply:
Sad. Suppose this was an oversight that just got missed somewhere along earlier. No reason to sink more time into it here with your update works just fine. Thank you for trying :)

Reply:
Yeah I think it comes back to the using context menu as the underlaying structure for the floating menu that makes things a bit more difficult.